### PR TITLE
2.5-alpha

### PR DESCRIPTION
**Pull request by daschuer**
This one introduces the new main branch version 2.5 alpha. 
After merge we can tag it accordingly and can merge out the 2.4 branch using the commit before. 

I have tracked the changes here that we can easily repeat them later: 
https://github.com/mixxxdj/mixxx/wiki/Mixxx-Beta-Checklist-2.4-beta

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## [2.5.0](https://launchpad.net/mixxx/+milestone/2.4.0) (Unreleased)
```
Comment (Swiftb0y) [inline CHANGELOG.md:3]: I guess since launchpad is unused for issue tracking, it rather makes sense to link to github milestones now, doesn't it?

**Comment by daschuer** [comment]
The build failure will be fixed after 
https://github.com/mixxxdj/mixxx/pull/10901
is merged and the 2.3 branch is merged to main 

It is really interesting how such CI failures always happen before a release :-/

**Review by ywwg — APPROVED** [review]

**Comment by Holzhaus** [comment]
the 2.4.0 changelog modification should be a separate commit (first commit in the branch), because that needs to go into 2.4, not 2.5

**Comment by daschuer** [comment]
The plan is to merge this to main and cut out the 2.4 branch at the previous commit. 
This is required to have not two builds with the same number.

**Comment by Holzhaus** [comment]
I know, and the 2.4 branch should already use the GitHub milestone link, not Launchpad.

**Comment by daschuer** [comment]
Ah OK I have reverted the change. The Changelog is incomplete anyway.

File: res/linux/org.mixxx.Mixxx.metainfo.xml [hunk 2]
```diff
@@ -86,7 +86,7 @@
     </screenshot>
   </screenshots>
   <url type="homepage">https://mixxx.org</url>
-  <url type="bugtracker">https://github.com/mixxxdj/mixxx/issues/</url>
+  <url type="bugtracker">https://bugs.launchpad.net/mixxx</url>
```
Comment (ywwg) [inline res/linux/org.mixxx.Mixxx.metainfo.xml:89]: wrong url for bug tracker

**Review by ywwg — CHANGES_REQUESTED** [review]

File: res/linux/org.mixxx.Mixxx.metainfo.xml (hunk 2 above)
Comment (ronso0) [reply res/linux/org.mixxx.Mixxx.metainfo.xml:89]: @daschuer wrong git rerere record? : )

Comment (daschuer) [reply res/linux/org.mixxx.Mixxx.metainfo.xml:89]: it seems like I have messed up the merge. I will do it again.

**Comment by daschuer** [comment]
Done

**Review by ywwg — APPROVED** [review]

**Pull request merged**